client/me/account-password/index.jsx

```diff
@@ -171,3 +172,8 @@ module.exports = React.createClass( {
 		);
 	}
 } );
+
+export default connect(
+	null,
+	dispatch => bindActionCreators( { errorNotice }, dispatch )
```

Comment:
`bindActionCreators` accepts an object or a function. In the case of only needing one actionCreator, should we just pass it directly instead of through the object syntax? `bindActionCreators( errorNotice, dispatch )`. Just a question about patterns, not a strong suggestion.